Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -368,6 +368,18 @@ public bool OnlyUseDefaultSpriteName
             return !string.IsNullOrEmpty(getSpriteNameAction);
         }
     }
+
+    /// <summary>
+    /// Whether the furniture has power or not.
+    /// </summary>
+    /// <returns>True if the furniture has power.</returns>
+    public bool HasPower
+    {
+        get
+        {
+            return PowerConnection == null || World.Current.PowerNetwork.HasPower(PowerConnection);
```

Comment:
ye.. I haven't changed the logic there. If furniture doesn't have PowerConnection, means there is no need to power it, so it has always power.